CHANGES.md

```diff
@@ -2,6 +2,20 @@ Changes by Version
 ==================
 Release Notes.
 
+0.4.0
+------------------
+
+### Features
+
+- Add `dashboard global` command with auto-refresh
+- Add `dashboard global-metrics` command
+- Add traces search
+- Refactor `metrics thermodynamic` command to adopt the new query protocol
```

Comment:
Is this for adopting v2 query protocol?

Reply:
yes

Reply:
Is the metrics query(besides thermodynamic) adopting the v2 already?

Reply:
no, will do that in the next version.